modules/ray_tracing/src/userobjects/RayTracingStudy.C

```diff
@@ -1437,11 +1419,6 @@ RayTracingStudy::verifyUniqueRayIDs(const std::vector<std::shared_ptr<Ray>>::con
                      ray->getInfo(),
                      "\n",
                      other_ray->getInfo());
-
-      mooseError("Duplicated shared pointers to the same local Ray were found ",
-                 error_suffix,
-                 "\n\n",
-                 ray->getInfo());
```

Comment:
same question for those two removal.
If you are just trying to increase coverage, because they are hard to reach with unit tests, I d rather they be turned to asserts than removed

Reply:
Same thing here - this is checked elsewhere